vendor/github.com/giantswarm/e2etemplates/pkg/e2etemplates/aws_operator_chart_values.go

```diff
@@ -17,6 +17,8 @@ const AWSOperatorChartValues = `Installation:
       Update:
         Enabled: ${GUEST_UPDATE_ENABLED}
     Name: ci-aws-operator
+    Infra:
```

Comment:
These changes should be proposed first to https://github.com/giantswarm/e2etemplates and then run here `dep ensure -update github.com/giantswarm/e2etemplates`, you shouldn't change `vendor` files directly.